### PR TITLE
end of an era. Another godless megaPR.

### PR DESCRIPTION
**Pull request by cdb-is-not-good**
SMGs have increased firerate, generally by 1 step(200rpm->300rpm. 300->600. etc)

Specop nerfed. No longer has SMG lossless full auto, reduced damage by 20% as a base and changes the name to reflect that this is not a big SMG....but rather a small carbine.

Explosive shotgun shells removed from naturally spawning in the rare-ammo pool. Explosive shells cost in the lathe increased by a factor of two.

Incendiaries cost raised by factor of 1.5

Magazines for SBAW that holdPayload raised to roughly be in line with the box. SBAW mag holds about a third as many rounds as a box, so costs about a third as much.

Cost of explosive grenades increased across the board, this includes EMPS, Frags, Blast.

A box of AMR standard cost 1650, but a handful of off-button was 2400...? This has been amended. A handful of AMR rounds(5 bullets) is now 2000, a box is 12,000

Adds kriosan alts for the marshal fullhelm and gasmask. 

Removes the laurels bio protection.

Absolutist wintercoat given a bit of stiffness to compensate for having okayish armor(15 melee, energy, bullet.)

Adds a pair of lightly armored pieces. A Jingasa(steel conical hat) and an Ashigaru(Armored apron). They have about as much protection as a leather jacket and are available in the loadout for 1 point each

Adds in several new pieces of headware. 2 straw hats, 4 sunhats, a tiara a headdress and 1 funky helmet

Adds a buncha new undershirts.

Some actual twit put swimsuits in the suit section despite the fact that they're uniforms, they've been moved to under/misc.

Adds a new swimsuit, 1 new gown(with an alt) and an alternate barmaid uniform(directly under the fantasy barmaid dress in the loadout).

In line with the shotgun explo-shell nerfs, removes the random spawns of the Novichok, removes the ability to purchase it via uplink and significantly raises its price via cargo.

To compensate for the fact that it is A. rare as hell B. incredibly difficult to use(slowdown, needs to be braced, slow to wield) the Fenrir has been given back its old 1.2 damage value. 

Fenrir has also been given 25% less recoil to *try* to make it less sillymode, it's still largely going to require bracing but should now at least be useable if modded

Kills handholding. Guns will no longer complain at you when fired without brace/wielding. Mostly, this is because these messages happening repeatedly was causing lag for some players(I can't confirm if this was due to the messages themselves, or because every single shot ran through an extra set of switches to determine what message to print/roll dice to see if it even prints a message). If people REALLY want, we can add them back later.

Hunting claw & Halberd now have bonus damage versus most mobs that you might think of as prey.

Hunters tengolo blade has 25% more damage vs its favored enemies.

hunting claw costs slightly changed. Plasteel cost raised to 6, steel cost removed, added a requirement for a cannibal fang.

Render claw significantly buffed, given higher AP similar to a dagger- this is a weapon that requires a relatively dangerous component, it needed *some* kinda benefit.

Explo shells had previously had their AOE reduced to 1(only the tile they hit), this turned out to not be the nerf it was intended to be and instead reduced the chance of collateral hits to 0. They now have AOE 2(1 tile around point of impact.)

Trees now require Robustness to cut down faster, not mec.

Taking a cutting from a tray is now faster by a decent bit

Comments out 5 sets of ears - Bunny Ears Rabbet. Bunny ears Rabbet alt, Bunny ears tall, fox ears alt and wolf ears taller. 

Why? because these ears were added in a PR that changed the size of the ears.dmi file, this caused all ears to misalign when folks were laying down. The code is left in should someone want to do the work of taking the five sets of ears, fixing their sprites and re-adding them. 

The old file has been saved for posterity as well, incase someone wants to find a code solution to the misalignment and go back to having the taller .dmi size

Adds a couple boxes of shotgun ammo to the Huntmasters office(for the emperess), also slightly reorganizes things.

Hunter crossbow bolts now have hunting bonus damage(basically extra damage versus any living thing that "isnt" a player). More buffs in line with making them better at hunting beasties, but not so much machines and players.

Peridaxon given a bit of balancing. Slightly less effective and no longer fixes braindamage(make Alkysine for that)

Carp sharks(big and small) now have a chance to drop Mephitic Fangs, which can be crafted into a new herbal remedy that works like a weak peridaxon albeit with a similar toxicity to carpotoxin. 

The remedy is created via lodge-craft. Bottle->tooth->poppy seed-> hammer. This produces 3 units of slurry which when heated will break down into high concentration venom, toxin and protein. Get rid of the toxin and protein, mix in 9 units of vodka for 10 units of tincture. Ingest(ideally in units of 5u or less) and make sure to keep dylovene on h and as it IS still quite poisonous

finally kills RIG storage units like i had planned to.

Sprite for the teeth credit to Liberty Landing.

Carp great whites given some more health

Toxins now affect you if ingested instead of only when injected, albeit less severely due to first pass metabolism.

Carp now properly drop their special drops when butchered.

Berserkers given a bit more health and a bit more melee armor.

Berserkers all made a bit more common

Recipe for hunting claw made a little more forgiving due to cannibal teeth being impossible to consistently procure. Now requires a horn from Sargoyles, which aren't horribly uncommon and are also all hardspawns that can be relatively easily accessed.

Recipe for bone rifle adjusted. Now requires a new drop from Nightmares

Added a chance for some monsters to drop a different pool of parts when butchered, currently chance is 50/50.

Nightmare stalkers are given such, they now drop either their chitin(used for making a recurve) or an elongated spur(for the bone rifle). sowwy for the codersprite on the spur. 

Croaker Lord tongue has a use, finally. Can be turned into a weapon mod for bows that provides more damage but makes the bow much slower to draw, similar to the Artificers guild mod albeit less powerful.

rig storage modules exist again but will still not spawn naturally(probably.)

adds a new message to warn the player when internal damage is being caused specifically by an infection. hopefuly this will result in fewer people ignoring internal damage warnings that turn out to be by infections that proceed to kill them dead. A

Spaceacillian now works on corpses, so as to make dealing with folks who die via infection less...impossible. 

Adds a handful of herbal teas that can be made with the usual collection of healing chem filled plants. Poppies, Tears, Hands, Vale and Plump helms - add 3 of the respective plant to 10u of water along with a thing of cinnamon then cook on low heat for about a minute for a concentrated herbal tea that other than working better overall *also* allows for some effects that normally don't work when ingesting(such chem purging from dylo plants and antibiotic from spaceacillian plants).

Additionally, fixes the carp fang mince recipe, it now requires the poppy tisane.

I did a good bit of tweaking for the balance of the tisanes to make them worth the time and effort of making but they probably will need some further tweaking.

New critters, Farwa. Exotic lil bipedal felinids typically kept as pets by the wealthy- or found ruining ecosystems due to their large litters and surprising effectiveness as hunters. But they're just so darn cute!
Currently not mapped anywhere because the map is locked up by anotehr PR :( 

okay maybe this time it wont completely shit the bed. Defib now checks if you have so much brain damage as to be functionally dead and if so prevents revival - this should fix the issue where folks were getting revived as half alive zombies.

Maybe fixes alkysine to actually fix brain injuries.

wolf ears(tall) fixed. 

Hustler has been brought back and reworked as a heavy shotgun for the shield. Sorta like a low-tier SBAW, it lacks autofire nor the ability to have drums but instead gets less recoil and more damage compared to the other shield shotguns 

Weapons available on their disk for now and comes fairly expensive(the stats and cost will likely need adjusting).

reduces the pricetag of hustler, Arithrazine can be used on corpses

Infections no longer spread after death. Whos idea was this even?

Removes a reference to the SBAW in the Hustler autoshotgun

fixes a busted sprite for the MAC smg

Bleeding rate reduced by 25%

mech "armor" raised by 1 step. This one is a lil weird so follow with. Rather than having armor represented the way it is for other things(melee=x bullet=x energy=x) it's instead represented via two values. one is 'armor' a number between 1 and 5 and the other absorbtion, a flat reduction to incoming damage.  Armor has been raised from 1-5 to 2-6, so everything gets a step higher armor, this means that a ripley for instance has twice as much armor, but a seraph has only 20% more. Absorbtion values not touched.

This has been paired with a 50% increase in exo health for all combat mechs. 

For now, that is all. This will likely need further tweaking especially for big boy mechs that're rarely seen like the Seraph and such, but should go a ways towards making mecha a bit less flimsy in the face of small arms fire.

gives the maid a lethal weapon(ppc) as was suggested a long time ago, such that it can actually be used during an emergency that isn't the very narrow(and rare) incident of 'a buncha people rioting without lethal intent'

Maybe fixes the issue with organs not giving infection warnings to host.

Organ pain warnings are now in green instead of red, so as to stand out vs attack and other red warnings.

Ameridien shooters radiation reduced to 15 from 25

Particle gun rads reduced to 20 from 30

BS no longer start with 2 saiginis in their armory(partly because holy god these things are tuned up), instead getting two buffalos.

Buffalo is no longer CO kit, replaced with the Reworked Hustler.

## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:

/:cl:

### CODE REVIEW

**Comment by Trilbyspaceclone** [comment]
design disk for the specop needs to be tweaked it says smg still

**Comment by cdb-is-not-good** [comment]
oops!

File: code/modules/clothing/head/misc.dm [hunk 1]
```diff
@@ -13,15 +13,15 @@
 /obj/item/clothing/head/laurel
 	name = "Laurel wreath"
 	icon_state = "laurel"
-	desc = "A round wreath made of connected branches and leaves of the bay laurel."
+	desc = "A round wreath made of connected branches and leaves of the bay laurel." /* It's a Laurel, it's not bio resistant.
```
Comment (Trilbyspaceclone) [inline code/modules/clothing/head/misc.dm:16]: remove the code not commit it out, its armor

File: code/modules/trade/datums/trade_stations_presets/1-common/hellcat.dm [hunk 2]
```diff
@@ -88,7 +88,7 @@
 		"Xanorath Smugged Gear" = list(
 			/obj/item/clothing/suit/space/void/merc/xanorath,
 			/obj/item/melee/energy/sword/pirate,
-			/obj/item/gun/projectile/gyropistol,
+			/obj/item/gun/projectile/gyropistol = good_data("Man-Opener", list(0, 2), 28000), //holster sized 'i win' gun, keep this expensive and rare.
```
Comment (Trilbyspaceclone) [inline code/modules/trade/datums/trade_stations_presets/1-common/hellcat.dm:91]: wrong name

Comment (cdb-is-not-good) [reply code/modules/trade/datums/trade_stations_presets/1-common/hellcat.dm:91]: The joke is that Voidwolves call it a man-opener even though that's not strictly its name

**Comment by Trilbyspaceclone** [comment]
No longer some

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 3]
```diff
@@ -1886,3 +1886,59 @@
...
+	nutriment_desc = list("bitter tea" = 1)
+	nutriment_amt = 1 //a lil bit from the leaves and plant solids.
+	bitesize = 5
+	preloaded_reagents = list("p_tea" = 10, "water" = 10)
+	matter = list(MATERIAL_BIOMATTER = 5)
+	cooked = TRUE
+
+/obj/item/reagent_containers/food/snacks/tear_tisane
+	name = "sun tear tisane"
+	desc = "A somewhat concentrated decoction of sun tears. A pleasantly sweet tea, it does a better job at aiding the healing of burns than simply chewing the tears raw.."
```
Comment (Trilbyspaceclone) [inline code/modules/reagents/reagent_containers/food/snacks.dm:1904]: 2 dots?

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 4]
```diff
@@ -1886,3 +1886,59 @@
...
+	nutriment_desc = list("tart tea" = 1)
+	nutriment_amt = 1
+	bitesize = 5
+	preloaded_reagents = list("mh_tea" = 10, "water" = 10)
+	matter = list(MATERIAL_BIOMATTER = 5)
+	cooked = TRUE
+
+/obj/item/reagent_containers/food/snacks/vale_tisane
+	name = "vale bush tisane"
+	desc = "A somewhat concentrated decoction of poppy flower. Not entirely pleasant tasting, and it leaves your mouth tingling- still, it functions as a more effective analgesic and vasodilator than simply chewing the tears themselves.."
```
Comment (Trilbyspaceclone) [inline code/modules/reagents/reagent_containers/food/snacks.dm:1926]: 2 dots and a random dash?

**Comment by Trilbyspaceclone** [comment]
Wait why are the so much better then advanced chemicals?

File: code/modules/mob/living/simple_animal/hostile/big.dm [hunk 5]
```diff
@@ -376,14 +379,14 @@
 	icon_dead = "slepnir_dead"
 	melee_damage_lower = 30
 	melee_damage_upper = 35
-	maxHealth = 350
-	health = 350
+	maxHealth = 550
+	health = 3550
```
Comment (Trilbyspaceclone) [inline code/modules/mob/living/simple_animal/hostile/big.dm:383]: wowers

**Comment by cdb-is-not-good** [comment]
> Wait why are the so much better then advanced chemicals?

I wouldn't call them super advanced. On testing they only seemed to be marginally better than just eating the fruit itself(probably because eating any chem is going to be slow) and require hydro+kitchen, which isnt a small ask.

File: code/modules/mob/living/simple_animal/hostile/big.dm (hunk 5 above)
Comment (cdb-is-not-good) [reply code/modules/mob/living/simple_animal/hostile/big.dm:383]: LLOL oops

File: code/modules/projectiles/guns/projectile/automatic/battlerifle.dm [hunk 6]
```diff
@@ -146,3 +146,42 @@
...
+	force = WEAPON_FORCE_PAINFUL
+	caliber = CAL_SHOTGUN
+	slot_flags = SLOT_BACK
+	load_method = SINGLE_CASING|MAGAZINE
+	mag_well = MAG_WELL_RIFLE
+	matter = list(MATERIAL_PLASTEEL = 30, MATERIAL_PLASTIC = 15, MATERIAL_DIAMOND = 10)
+	gun_parts = null
+	price_tag = 3000
+	penetration_multiplier = 1.1
+	damage_multiplier = 1.3 //better than the saigini but no full auto
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/projectile/automatic/battlerifle.dm:170]: Why does this deal more then SBAW if its bad?

Comment (Trilbyspaceclone) [reply code/modules/projectiles/guns/projectile/automatic/battlerifle.dm:170]: Wait hold on this is amazingly better then the SBAW

Comment (cdb-is-not-good) [reply code/modules/projectiles/guns/projectile/automatic/battlerifle.dm:170]: More  dam mod less AP mod

Comment (cdb-is-not-good) [reply code/modules/projectiles/guns/projectile/automatic/battlerifle.dm:170]: though it is very weird to me that the SBAW lost its damage mod ages ago

Comment (cdb-is-not-good) [reply code/modules/projectiles/guns/projectile/automatic/battlerifle.dm:170]: > Wait hold on this is amazingly better then the SBAW

How so? Less AP, can't take drums, slows you down when wielded, less modding, more expensive(though less rare)

**Comment by Trilbyspaceclone** [comment]
Please move the map changes to a different pr
Please move the guns tweaks/added news ones to a different pr
Please move the teas/carp to a different pr

**Comment by cdb-is-not-good** [comment]
> Please move the map changes to a different pr Please move the guns tweaks/added news ones to a different pr Please move the teas/carp to a different pr

augghhhhhhhhhhgghgggghgh but then it'd have to wait LONGER before being merged. I'll let this PR lie and stop adding new stuff to it

**Comment by cdb-is-not-good** [comment]
3ish days since last major commit. good enough

**Pull request merged**